inc/inventory.class.php

```diff
@@ -58,9 +58,11 @@ public static function updateInventory($params = [])
                     if ($itemtype == Computer::getType()) {
                         //load inventory from DB because
                         //FI not update XML file if computer is not update
-                        /** @phpstan-ignore-next-line  */
+                        // @phpstan-ignore-next-line
                         $db_info = new PluginFusioninventoryInventoryComputerComputer();
+                        // @phpstan-ignore-next-line
                         if ($db_info->getFromDBByCrit(['computers_id' => $items_id])) {
+                            // @phpstan-ignore-next-line
```

Comment:
You need to test the existence of `PluginFusioninventoryInventoryComputerComputer`